src/Dfe.EarlyYearsQualification.Web/Helpers/PlaceholderUpdater.cs

```diff
@@ -0,0 +1,25 @@
+namespace Dfe.EarlyYearsQualification.Web.Helpers;
+
+public class PlaceholderUpdater : IPlaceholderUpdater
+{
+    private readonly Dictionary<string, string> _placeholders = new()
+                                                                {
+                                                                    { "$[actual-year]$", DateTimeOffset.Now.Year.ToString() }
```

Comment:
Suggestion (minor). Single responsibility: I'd inject an `IDateTimeAdapter` instance into this service, and get the year from that? That might make unit testing easier, particularly if we ever need to extend the functionality to other placeholders. _This_ service's responsibility is only to replace placeholders. Actually, we could then replace this dictionary with a static array of placeholders.  🙂

Reply:
Injected the adapter now :-)